samples/bluetooth/central_otc/src/main.c

```diff
@@ -168,14 +175,14 @@ static void configure_button_irq(const struct gpio_dt_spec btn)
 	}
 
 	ret = gpio_pin_configure_dt(&btn, GPIO_INPUT);
-	if (ret != 0) {
+	if (ret) {
```

Comment:
Why are you making these changes? Even though the Bluetooth stack doesn't fall under the safety certification scope, we should still try to follow the Zephyr coding guidelines which strive for MISRA compliance. One of the requirements is that integers should not be used as booleans in conditional statements.

Reply:
I saw many of Zephyr sample use integers that way. Trying to align to those sample.

Reply:
@pirun I suggest we remove those changes to expedite the merging of this PR. It makes the diff smaller as well, which is nice.

Reply:
@jhedberg @alwa-nordic Agreed. Thanks for your opinions.

Reply:
Yes, I'm aware of inconsistencies. There's a lot of code in the tree that predates the safety certification related coding guidelines. A lot of that code hasn't been updated, and then there's probably new code that took example from the old code. FWIW, even if you'd have had these changes in this PR you shouldn't mix them into a commit that contains an unrelated functional change, rather style changes, etc should be in separate commits.

Reply:
For reference, I'm talking about MISRA C:2012 Rule 14.4. Here are some PRs which fix code related to it: #48070 #48082 #48065 #40415

Reply:
I'm wondering if it is okay to add additional commit to fix style in this PR?